pkg/reconciler/dependencybuild/dependencybuild.go

```diff
@@ -559,9 +565,37 @@ func (r *ReconcileDependencyBuild) handleBuildPipelineRunReceived(ctx context.Co
 
 		success := pr.Status.GetCondition(apis.ConditionSucceeded).IsTrue()
 		if success {
+			for _, i := range pr.Status.PipelineResults {
```

Comment:
ah ok so the java writes to the results, and the golang controller consumes and updates our CRD instances' statuses 

sure, I think that make sense now when I revisit our strategizing discussions :-)

Reply:
It's the only way I can think of to actually propagate this information back without involving some form of external service.

Reply:
yep that's right